integration_tests/inttest_lib/runners/lambda_function_runner.py

```diff
@@ -74,8 +75,10 @@ def _is_lambda_completed(self, lambda_function_name, request_id, start_time):
         query_string = f"fields @timestamp, @message | filter @message like /END RequestId: {request_id}/"
 
         try:
+            query_start_time = int(datetime_to_epoch_milliseconds(start_time))
```

Comment:
can you please double-check it doesn't break LambdaFunctionRunner in integration tests (it also uses query_logs)